statsmodels/compat/pandas.py

```diff
@@ -0,0 +1,11 @@
+import pandas
```

Comment:
My suggestion didn't work.  Instead you can do this

in `compat.init.py`

```
from __future__ import absolute_import
from distutils.version import LooseVersion

from statsmodels.compat.python import *
import pandas as pd

pandas_version = LooseVersion(pd.__version__)
```
in `compat.pandas`
```
import statsmodels.compat

version = statsmodels.compat.pandas_version

if version >= '0.17.0':
    def sort_values(df, *args, **kwargs):
        return df.sort_values(*args, **kwargs)
else:
    def sort_values(df, *args, **kwargs):
        return df.sort(*args, **kwargs)
```